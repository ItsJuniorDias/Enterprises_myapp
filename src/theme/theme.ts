export const theme = {
  colors: {
    primary: '#376aed',
    secondary: '#0d253c',
    light_secondary: '#2d4379',
    error: '#ef233c',
    skeleton: '#adb5bd',
    shape: '#FFFFFF',
    text: '#000000',
  },
  fonts: {
    regular: 'Poppins-Regular',
    semiBold: 'Poppins-SemiBold',
    bold: 'Poppins-Bold',
    medium: 'Poppins-Medium',
  },
};

export type ThemeType = typeof theme;
