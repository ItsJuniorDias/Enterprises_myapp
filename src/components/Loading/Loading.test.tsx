import React from 'react';
import { render } from '@testing-library/react-native';

import { Loading } from '../Loading/Loading';
import { ThemeProvider } from 'styled-components/native';
import { theme } from '../../theme';

describe('Behavior Loading', () => {
  const screenRender = (
    <ThemeProvider theme={theme}>
      <Loading />
    </ThemeProvider>
  );

  it('render snapshot', () => {
    const result = render(screenRender).toJSON();

    expect(result).toMatchSnapshot();
  });
});
