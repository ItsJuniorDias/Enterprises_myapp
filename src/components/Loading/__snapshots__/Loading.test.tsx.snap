// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Behavior Loading render snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flexBasis": 0,
      "flexGrow": 1,
      "flexShrink": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    color="#2D4379"
    size="large"
  />
</View>
`;
